README.md

```diff
@@ -198,7 +199,47 @@ The library adds a `form-submitted` to the submitted form. You can use it to sty
   }
 }
 ```
+- `controlErrorComponent` - Optional. Allows changing the default component that is used to reflect 
+  the errors. This component should implement the `IControlErrorComponent` interface.  Defaults to
+  `ControlErrorComponent`.
+
+  For example:
+  ```ts
+  {
+      controlErrorComponent: CustomControlErrorComponent
+  }
+  ```
+
+  `CustomControlErrorComponent` is declared as:
+  ```ts
+    @Component(...)
```

Comment:
Please add a complete example and use Ionic as a reference.

Reply:
Good idea. Will do.